kolibri/core/assets/src/views/StorageNotification.vue

```diff
@@ -47,53 +47,80 @@
 <script>
 
   import commonCoreStrings from 'kolibri.coreVue.mixins.commonCoreStrings';
-  import useUser from 'kolibri.coreVue.composables.useUser';
   import { mapGetters } from 'vuex';
+  import useUser from 'kolibri.coreVue.composables.useUser';
+  import { useLocalStorage } from '@vueuse/core';
+  import { ref } from 'kolibri.lib.vueCompositionApi';
+  import { LearnerDeviceStatus } from 'kolibri.coreVue.vuex.constants';
+  import useUserSyncStatus from '../composables/useUserSyncStatus';
 
   export default {
     name: 'StorageNotification',
     components: {},
     mixins: [commonCoreStrings],
     setup() {
+      const local_storage_last_synced = ref(useLocalStorage('last_synced', ''));
+      const local_storage_lastDownloadRemoved = ref(useLocalStorage('last_download_removed', ''));
```

Comment:
Looking at the `vueuse` docs, it appears the return value from `useLocalStorage` is already a `Ref`. The [docs](https://v9-13-0.vueuse.org/core/useLocalStorage/) show it returns `RemovableRef` under 'Type Declarations':
```ts
export declare function useLocalStorage(
  key: string,
  initialValue: MaybeRefOrGetter<string>,
  options?: UseStorageOptions<string>
): RemovableRef<string>
```

The ending part `: RemovableRef<string>` says `useLocalStorage` returns the `RemovableRef` type, which can hold values of type `string` (the part inside `<` `>`)

Admittedly, it was hard to track whether `RemovableRef` is a sub-class/type of `Ref`, but it appears so:
https://github.com/vueuse/vueuse/blob/bddfbae423426ea70a2cf961350562c2b7758cd6/packages/shared/utils/types.ts#L13-L19

Reply:
Based on @AllanOXDi's request, I will be stepping in to resolve any pending issues on this pr while he is away.